# [LAB-02] 4. 기술 통계량의 이해 (1)

## 연습문제 1

In [ ]:
import numpy as np

alpha = [88, 92, 76, 100, 95, 81, 79, 98, 85, 87]
brabo = [65, 99, 78, 72, 80, 91, 75, 82, 89, 70]

print("==== Alpha팀 기술통계량 ====")
print("도수:", len(alpha))
print("평균:", np.mean(alpha))
print("표준편차:", np.std(alpha))
print("최소값:", np.min(alpha))
print("1사분위 수:", np.quantile(alpha, 0.25))
print("중앙값:", np.quantile(alpha, 0.5))
print("3사분위 75:", np.quantile(alpha, 0.75))
print("최대값:", np.max(alpha))

print("\n==== Brabo팀 기술통계량 ====")
print("도수:", len(brabo))
print("평균:", np.mean(brabo))
print("표준편차:", np.std(brabo))
print("최소값:", np.min(brabo))
print("1사분위 수:", np.quantile(brabo, 0.25))
print("중앙값:", np.quantile(brabo, 0.5))
print("3사분위 75:", np.quantile(brabo, 0.75))
print("최대값:", np.max(brabo))

==== Alpha팀 기술통계량 ====
도수: 10
평균: 88.1
표준편차: 7.7
최소값: 76
1사분위 수: 82.0
중앙값: 87.5
3사분위 75: 94.25
최대값: 100

==== Brabo팀 기술통계량 ====
도수: 10
평균: 80.1
표준편차: 9.924212815130478
최소값: 65
1사분위 수: 72.75
중앙값: 79.0
3사분위 75: 87.25
최대값: 99


## 연습문제 2

In [ ]:
import numpy as np

traffic_acc = np.array([15129, 13836, 15738, 16300, 17605, 16690, 16252, 17000, 16956, 17720, 17455, 15668])

print("도수:", len(traffic_acc))
print("평균:", np.mean(traffic_acc))
print("표준편차:", np.std(traffic_acc))
print("최소값:", np.min(traffic_acc))
print("1사분위 수:", np.quantile(traffic_acc, 0.25))
print("중앙값:", np.quantile(traffic_acc, 0.5))
print("3사분위 75:", np.quantile(traffic_acc, 0.75))
print("최대값:", np.max(traffic_acc))

도수: 12
평균: 16362.416666666666
표준편차: 1089.3134273732037
최소값: 13836
1사분위 수: 15720.5
중앙값: 16495.0
3사분위 75: 17113.75
최대값: 17720


### 2024년 월별 교통사고(traffic_acc) — 통계 기반 인사이트

#### 1) 중심 경향성 — 평균과 중앙값의 관계

- 평균(≈16,362)과 중앙값(16,495)은 매우 근접합니다(차이 ≈ -133). 이로 미루어 전체 분포의 중심은 한 군데에 모여 있으며, 평균이 중앙값보다 작다는 점은 왼쪽으로 약간 긴 꼬리가 존재함을 시사합니다. 이는 데이터가 오른쪽으로 쏠림 현상이 있다는 것을 의미합니다.

#### 2) 변동성(표준편차)과 상대적 불확실성

- 표준편차가 약 1,089로 평균 대비 약 6.7% 수준입니다. 즉 월별 사고 수는 평균을 중심으로 대체로 ±1,000건 내외의 변동을 보이고 있습니다.

#### 3) 사분위수 판단

Q3(≈17,113.75) 이상인 달은 상위 25%로, 주어진 데이터에서는 3개(예: 17,720, 17,605, 17,455 / 5, 10, 11월)가 여기에 해당합니다.

이들 달을 우선적으로 원인 분석(기상·통행량·행사·도로조건 등)하고 예방책을 집중 배치하면 효과적입니다.

12월은 상대적으로 술자리가 많은 시즌이므로 많은 운전자들이 운전을 자제하는 것으로 보입니다.

또한 1월은 눈이 많이 오는 계절이기 때문에 운전을 할 수 없는 경우가 많은 것으로 보입니다.

#### 참고자료

- 11월 고속도로 교통사고 사망자 ‘연중 최다’…2차 사고 주의보 [출처:중앙일보]
  - https://www.joongang.co.kr/article/25379158
- 단풍놀이 가다가…10월 교통사고 사망↑ [출처:KBS]
  - https://news.kbs.co.kr/news/pc/view/view.do?ncd=8384587
- 초겨울인 11월 안개 교통사고 주의！
  - https://www.koroad.or.kr/main/board/6/7476/board_view.do?&cp=25&listType=list&bdOpenYn=Y&bdNoticeYn=N